apps/telehealth-widget/README.md

```diff
@@ -0,0 +1,323 @@
+# Amazon Chime SDK Telehealth Widget Demo
+
+The Amazon Chime SDK Telehealth Widget demo demonstrates various communication capabilities provided by Amazon Chime SDK. The demo showcases patient-doctor use cases with below user scenarios:
+
+|User scenario|Service flow|
+|--|--|
+|A doctor and a patient sign up and sign in to the widget. |The solution uses [Amazon Cognito](https://aws.amazon.com/cognito/) for authentication and authorization. When the user signs up, the solution creates an Amazon Chime SDK `AppInstanceUser`.
+|A doctor creates an appointment with a patient. |The solution creates an [Amazon Chime SDK messaging](https://docs.aws.amazon.com/chime-sdk/latest/dg/using-the-messaging-sdk.html) channel and adds two users as `ChannelModerator`. 
+|A doctor and a patient view current and upcoming appointments. |The solution lists Amazon Chime SDK messaging channels moderated by the user.
+|A doctor and a patient check in for an appointment. |The solution uses [Amazon Chime SDK messaging channel flows](https://docs.aws.amazon.com/chime/latest/dg/using-channel-flows.html) to set the user's presence status to "Checked in."
+|A doctor and a patient join a video call. |The solution initiates an [Amazon Chime SDK meeting](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html) with two Amazon Chime SDK attendees.
+|A patient receives an SMS notification if a patient has not checked in. |The solution uses [Amazon Pinpoint](https://aws.amazon.com/pinpoint/) to send an SMS notification.|
+|A doctor and a patient join a meeting with a PSTN call. |The solution uses [Amazon Chime SDK PSTN Audio service](https://docs.aws.amazon.com/chime-sdk/latest/dg/build-lambdas-for-sip-sdk.html) to make a PSTN outbound call from doctor to patient.|
+
+## Demo screenshots
```

Comment:
How about moving the **Demo screenshots** and **Solution Architecture** sections to the bottom as Appendix?

The solution architecture contains all the details, but I feel that only a few readers will understand the listed AWS services and flows.

- Getting started
- ...
- Appendix A: Solution architecture
- Appendix B: Demo screenshots